src/ast/ast_mapping.c

```diff
@@ -122,12 +122,15 @@ static void _AST_MapExpression(AST *ast, const cypher_astnode_t *expr) {
 		return;
 	} else if(type == CYPHER_AST_PROC_NAME) {
 		return;
-	} else if(type == CYPHER_AST_INTEGER ||
-			  type == CYPHER_AST_FLOAT   ||
-			  type == CYPHER_AST_STRING  ||
-			  type == CYPHER_AST_TRUE    ||
-			  type == CYPHER_AST_FALSE   ||
-			  type == CYPHER_AST_NULL) {
+	} else if(type == CYPHER_AST_INTEGER            ||
+			  type == CYPHER_AST_FLOAT              ||
+			  type == CYPHER_AST_STRING             ||
+			  type == CYPHER_AST_TRUE               ||
+			  type == CYPHER_AST_FALSE              ||
+			  type == CYPHER_AST_NULL               ||
+			  type == CYPHER_AST_COLLECTION         ||
```

Comment:
I think there are cases in which we would want to map the expressions contained in a CYPHER_AST_COLLECTION (using the logic applied to CYPHER_AST_COMPARISON above).

Reply:
lets talk about it on Monday